### DISCUSSION TITLE
Awesome Akash needs to be updated so it can stay Awesome

### DISCUSSION THREAD

**Discussion by oiclid** [akash-network/community#675] [General]
While working on the Akash documentation, I took a deep dive into the Awesome Akash repo. During this process, I realized that it is in serious need of reorganization. Many of the entries are old or out-of-date, with some pointing to projects or software that no longer exist or haven't been updated or supported for years.

To ensure that the Awesome Akash repo remains a valuable resource for the community, I plan to thoroughly review each entry. My goal is to test, edit, update, add, or delete SDLs as necessary. I'll be leveraging the Testnet/Sandbox for this task, allowing me to do so with minimal cost.

Additionally, I aim to introduce new deployments and SDLs that are relevant to the current state of Akash. This will ensure that the list stays comprehensive and up-to-date, making it a go-to resource for anyone looking to deploy on Akash.

It's a comprehensive task, but I'm committed to updating every single item to reflect the best and most current information available.

**Comment by piber-dev** [comment]
this is a great idea but i am worried that we are going to double or triple the amount of work needed with awesome-akash. earlier this year, i was talking to tyler, vanguards, and insiders and was promised a $750 bounty of reorganizing awesome-akash and ended up with [this fork](https://github.com/piber-dev/awesome-akash) but it was never merged nor did i get the bounty or any feedback when tyler attempted to get it. there's 61 commits that needs to be manually added to this if it should be merged before it makes sense to do this big task you want to do. maybe lets plan to discuss everything related to awesome-akash in a wg/sig meeting someday?

**Reply by brewsterdrinkwater** [reply]
I second the thoughts of Piber. He has already worked on this effort, and there is a fork that he shared.

I believe others should look at the work from Piber, and give feedback inside of Discord. This can be done in the sig-clients channel. It would also be great to hear from different audiences of Akash users on how they currently use Awesome-Akash. 

Awesome-Akash is also used by Akash Console for templates, so we want to take that into account as well.

TLDR: Look at Piber's PR and leave comments, so that this discussion can get more traction, and necessary updates can be made.

P.S.: when the changes are merged, Piber will be paid the full bounty,as outlined.

**Reply by piber-dev** [reply]
thanks for the reply @brewsterdrinkwater - i believe the only change to make it compatible with akash console is to change add `/templates/` as a suffix to the place where templates are loading from akash console - a 5 minute fix for anyone familiar with the console codebase.

**Reply by oiclid** [reply]
That makes sense. I'll share notes I have when appropriate. @piber-dev I did look at your repo, and the reorganization looks great!

**Reply by oiclid** [reply]
@brewsterdrinkwater What about the new SDLs? I already created and tested some for the guides, and was thinking of creating a few more. Should I also suspend doing them?

**Reply by baktun14** [reply]
Hey @piber-dev, this is great, but it would break console at the templates are now under a folder(s) and categories are now different. Also, the archive is confusing, why keep it? I think it would be better to just remove the ones that don't work.
